content/en/blog/_posts/2022-12-08-kubernetes-1.26-blog.md

```diff
@@ -0,0 +1,354 @@
+feature flag there will be an additional metrics endpoint which allows the calculation of Service
+Level Objectives (SLOs) from raw healthcheck data converted into metric format.
+
+#### Feature metrics are now available
+
+Feature metrics are now available for each Kubernetes component, making it possible to [track
+whether each active feature gate is enabled](https://github.com/kubernetes/kubernetes/pull/112690)
+by checking the component's metric endpoint for `kubernetes_feature_enabled`.
+
+### Dynamic Resource Allocation graduates to alpha
+
+[Dynamic Resource
+Allocation](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/)
+is a [new feature](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/3063-dynamic-resource-allocation/README.md) 
+that puts resource scheduling in the hands of third-party developers: it offers an
+alternative to the limited "countable" interface for requesting access to resources
+(e.g. `nvidia.com/gpu: 2`), providing an API more akin to that of persistent volumes. Under the
+hood, it uses the [Container Device
+Interface](https://github.com/container-orchestrated-devices/container-device-interface) (CDI) to do
+its device injection. This feature is blocked by the `DynamicResourceAllocation` feature gate.
+
+### CEL in Admission Control graduates to alpha
+
+[This feature](https://github.com/kubernetes/enhancements/issues/3488) introduces a `v1alpha1` API for [validating admission
+policies](/docs/reference/access-authn-authz/validating-admission-policy/), enabling extensible admission
+control via [Common Expression Language](https://github.com/google/cel-spec) expressions. Currently,
+custom policies are enforced via [admission
+webhooks](docs/reference/access-authn-authz/extensible-admission-controllers/),
+which, while flexible, have a few drawbacks when compared to in-process policy enforcement. To use,
+enable the `ValidatingAdmissionPolicy` feature gate and the `admissionregistration.k8s.io/v1alpha1`
+API via `--runtime-config`.
+
+### Pod scheduling improvements
+
+Kubernetes v1.26 introduces some relevant enhancements to the ability to better control scheduling
+behavior.
+
+#### `PodSchedulingReadiness` graduates to alpha
+
+[This feature](https://github.com/kubernetes/enhancements/issues/3521) introduces a `.spec.schedulingGates` 
+field to Pod's API, to [indicate whether the Pod is allowed to be scheduled or not](/docs/concepts/scheduling-eviction/pod-scheduling-readiness/). External users/controllers can use this field to hold a Pod from scheduling based on their policies and
+needs.
+
+#### `NodeInclusionPolicyInPodTopologySpread` graduates to beta
+
+By specifying a `nodeInclusionPolicy` in `topologySpreadConstraints`, you can control whether to
+[take taints/tolerations into consideration](/docs/concepts/scheduling-eviction/topology-spread-constraints/)
+when calculating Pod Topology Spread skew.
+
+#### `MatchLabelKeysInPodTopologySpread` graduates to beta
```

Comment:
Sorry, this is still in alpha. https://github.com/kubernetes/enhancements/issues/3243, cc @alculquicondor for confirm.

Reply:
that is correct, this is still alpha.

Reply:
Isn't this this one https://github.com/kubernetes/enhancements/issues/3094 ? It is on the tracking board as graduating to beta.

Reply:
They're not the same, https://github.com/kubernetes/enhancements/issues/3094 is graduated to beta in this release, but not `MatchLabelKeysInPodTopologySpread`.

Reply:
[updated the stage](https://github.com/kubernetes/enhancements/issues/3094#issuecomment-1342837670).. thanks @kerthcet for your review!

Reply:
https://github.com/kubernetes/enhancements/issues/3243 is not part of the release

Reply:
```suggestion
#### Improved pod topology spread constraints
```
?

Reply:
This was part of the [major themes discussion](https://github.com/kubernetes/sig-release/discussions/2047#discussioncomment-4162653) and was described as beta there, and as a major theme.  It was removed from milestone though, so I think this is a case where that wasn't captured. I would simply remove it since there is no movement in 1.26 related to it.

Reply:
Done in last commit, will squash if there is no regression needed.